<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistanceWidget</class>
 <widget class="QDockWidget" name="DistanceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distance Between Point Set and Mesh</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Statistics</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="7" column="1">
          <widget class="QLabel" name="maxLabel">
           <property name="text">
            <string>max</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="lDecildeLabel">
           <property name="text">
            <string>last decile</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Minimum Distance:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Last Decile:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="fDecileLabel">
           <property name="text">
            <string>first decile</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="minLabel">
           <property name="text">
            <string>min</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Maximum Distance:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>First Decile:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Median Distance:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="medianLabel">
           <property name="text">
            <string>median</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QPushButton" name="pushButton">
           <property name="toolTip">
            <string>Creates a new colored point set. The further a point is from the mesh, the darker it is displayed.</string>
           </property>
           <property name="text">
            <string>Compute Distance</string>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>minLabel</zorder>
        <zorder>fDecileLabel</zorder>
        <zorder>lDecildeLabel</zorder>
        <zorder>label</zorder>
        <zorder>label_3</zorder>
        <zorder>label_4</zorder>
        <zorder>label_2</zorder>
        <zorder>maxLabel</zorder>
        <zorder>label_5</zorder>
        <zorder>medianLabel</zorder>
        <zorder>pushButton</zorder>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Selection</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
           <item row="1" column="1">
            <widget class="QPushButton" name="select_button">
             <property name="toolTip">
              <string>Selects the points that are further than the threshold.</string>
             </property>
             <property name="text">
              <string>Select </string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Threshold:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="DoubleEdit" name="distance_spinbox">
             <property name="text">
              <string>0.00</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>CGAL_double_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
